### PR TITLE
Added Landing page routes and authentication

### PR DESCRIPTION
**Pull request by Amandeep-sinha**
Added :- 
Landing Page 
public Routes
private routes
Authentication

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+.DS_Store
+
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
```
Comment (vaibhav18matere) [inline .gitignore:24]: you should not push `gitignore` files to github.

File: src/backend/controllers/ArchiveController.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+/**
+ * All the routes related to Archives are present here.
+ *  These are Privately accessible routes.
+ * */
+
+/**
+ * This handler handles gets all archived notes in the db.
+ * send GET Request at /api/archives
+ * */
```
Comment (vaibhav18matere) [inline src/backend/controllers/ArchiveController.js:12]: remove comments

**Review by vaibhav18matere — APPROVED** [review]
looks good to merge

File: .gitignore (hunk 1 above)
Comment (Amandeep-sinha) [reply .gitignore:24]: okay will take care from next time.

**Pull request merged**